.github/workflows/CI.yml

```
@@ -63,9 +63,13 @@ jobs:
               rustup toolchain add nightly-x86_64-unknown-linux-gnu
               rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
           # Disabled for now
-          #- runner: ubuntu-latest
-          #  target: aarch64-unknown-linux-musl
-          #  cross-compile: true
+          - runner: ubuntu-24.04-arm
+            target: aarch64-unknown-linux-musl
+          - runner: ubuntu-24.04-arm
```

actionlint (1.7.4)

duplicate value {"cross-compile": "true", "runner": "ubuntu-24.04-arm", "target": "aarch64-unknown-linux-musl"} is found in matrix "environments". the same value is at line:66,col:13

(matrix)